Add abi_features to sys (#1)
zklaus/cpython: Doc/library/sys.rst

always available. Unless explicitly noted otherwise, all variables are read-only.

.. data:: abi_features

>    A frozen set of ABI features.

Reviewer: ```suggestion
   A frozen set of strings that represent ABI features.
```

   Currently supported features are:

   ``free-threading`` or ``gil-enabled``
      The Python interpreter is free-threaded. On POSIX systems, this corresponds
      to ``"t" in sys.abiflags``.

   ``debug``
      The Python interpreter has debugging capabilities. On POSIX systems, this
      corresponds to ``"d" in sys.abiflags``.

   ``32-bit`` or ``64-bit``
      The bitness of the interpreter, that is, whether it is a 32-bit or 64-bit

>       build.

Reviewer 2: I don't think this is the right place. This information is part of the target architecture, where this API is meant to expose features of the Python ABI — see [PEP 3149](https://peps.python.org/pep-3149/).

always available. Unless explicitly noted otherwise, all variables are read-only.

.. data:: abi_features

   A frozen set of ABI features.
   Currently supported features are:

>    ``free-threading``

Reviewer 3: ```suggestion
   ``free-threading`` or ``gil_enabled``
```

>       The Python interpreter is free-threading. On Unix systems, this corresponds

Reviewer 3: ```suggestion
      The Python interpreter is free-threaded. On POSIX systems, this corresponds
```

      to ``"t" in sys.abiflags``.

   ``debug``

>       The Python interpreter has debugging capabilities. On Unix systems, this

Reviewer 3: ```suggestion
      The Python interpreter has debugging capabilities. On POSIX systems, this
```

      corresponds to ``"d" in sys.abiflags``.

   ``32-bit`` or ``64-bit``
      The bitness of the interpreter, that is, whether it is a 32-bit or 64-bit
      build.

   .. versionadded:: 3.14

>    .. availability:: Unix, Windows

Reviewer 3: No need for an `.. availability` directive, if it's available everywhere. Also, Unix & Windows is not an exhaustive list.
